app/api/schema.py

```diff
@@ -1,5 +1,15 @@
-from graphene import AbstractType
+from .cruds.user_crud import UserNode, CreateUser, UpdateUser, DeleteUser
```

Comment:
@andela-aoduola: Since this is import from custom module, it's most appropriate to be below import from any django module.